### PR TITLE
Hyperlink teacher list in manage class page

### PR DESCRIPTION
**Pull request by betaveros**
This makes it easier to do a variety of tasks involving finding teachers
based on what classes they're teaching

### CODE REVIEW

File: esp/esp/users/models/__init__.py [hunk 1]
```diff
@@ -214,6 +214,11 @@ def ajax_str(self):
     def name(self):
         return u'%s %s' % (self.first_name, self.last_name)
 
+    def nonblank_name(self):
+        name = self.name()
+        if name.strip() == '': name = self.username
```
Comment (betaveros) [inline esp/esp/users/models/__init__.py]: I don't know how necessary this defensive logic really is; I copied it from `getTeacherNames`

File: esp/templates/program/modules/adminclass/manageclass_class_info.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <tr>
     <th class="smaller">Teachers:</th>
-    <td>{{ class.getTeacherNames|join:", " }}<br />
+	<td>{% for teacher in class.get_teachers %}<a href="/manage/userview?username={{ teacher.username|urlencode }}">{{ teacher.nonblank_name }}</a>{% if not forloop.last %}, {% endif %}{% endfor %}
```
Comment (betaveros) [inline esp/templates/program/modules/adminclass/manageclass_class_info.html]: The correct way to do this definitely involves `{% url something %}`, but I really want these links for sub wrangling and the rest of the codebase uses this URL creation method *a lot* so I'm going to keep this until after Splash at least

Comment (benjaminjkraft) [inline esp/templates/program/modules/adminclass/manageclass_class_info.html]: Yeah, we don't really use `{% url %}` unfortunately, so this is fine.

File: esp/esp/users/models/__init__.py (hunk 1 above)
Comment (benjaminjkraft) [inline esp/esp/users/models/__init__.py]: I'm not sure it's necessary, but it doesn't really hurt.

**Pull request merged**